scripts/test/validate-build.ts

```diff
@@ -335,7 +335,7 @@ async function validateModuleTreeshake(opts: BuildOptions, moduleName: string, e
 
   const bundle = await rollup({
     input: virtualInputId,
-    treeshake: true,
+    treeshake: 'smallest',
```

Comment:
A change was [introduced in rollup@2.52.0](https://github.com/rollup/rollup/blob/master/CHANGELOG.md#2520) which adds preset options that can be set here, see the [PR which introduced the options](https://github.com/rollup/rollup/pull/4131) for more on this.

Without this change I get a build error when doing `node scripts/build --validate-build`